Header header
# geometry_msgs/Point position
# geometry_msgs/Vector3 velocity
# geometry_msgs/Vector3 acceleration
# geometry_msgs/Vector3 jerk
# float64 yaw
# float64 yaw_dot
rpg_mpc/PositionCommand[] points
geometry_msgs/Point state_posi
geometry_msgs/Vector3 state_vel
geometry_msgs/Quaternion state_q